xbmc/pvr/PVRGUIInfo.cpp

```diff
@@ -711,7 +711,7 @@ void CPVRGUIInfo::UpdateBackendCache(void)
 
   {
     CSingleLock lock(m_critSection);
-    if (m_iAddonInfoToggleCurrent >= iActiveClients)
+    if (m_iAddonInfoToggleCurrent >= static_cast<unsigned int>(iActiveClients))
```

Comment:
Could you drop this one? We have a PR pending which gets rid of this variable completely.